### ISSUE TITLE
Newsletter section

### ISSUE THREAD

**Issue by altonwebsites** [altonwebsites/EBPP#18]
There is no link provided to take the user to the “Newsletter” page, and the actual “Newsletter” page has no content. 
Can we migrate all the past newsletters here?

**Comment by altonwebsites** [comment]
Newsletters page with content on the way. Didn't have time to add them pre-Christmas

**Comment by altonwebsites** [comment]
I've created the Newsletter page as requested in the 13.10.14 Website Redesign Word Doc referring to http://www.balieye.org/news.htm :
"The News page is good because it includes all downloadable PDF versions of the newsletter with the most recent one at the top with an introductory paragraph. It would be good to have something similar for EBPP’s Newsletters, press releases, annual report where we simply provide a link with month/year as the title."

In the future, to add new PDF Newsletters to the site use the Newsletter menu item on the right hand side and select Add New. 

The title and text content fields should will be familiar. 

The Attachments field allows multiple newsletter versions to be uploaded - for the different language versions I've seen in the archives. If text is added to the Attachment Description field, e.g. "English version", "Indonesian version" the web page will replace "Click to download the Newsletter" with "Click to download the English version" or "Click to download the Indonesian version" 

WordPress doesn't have a way to create an image from the PDF so the Cover Image is created using the website: http://pdf2jpg.net/

I've added only the three newsletters that weren't archived, it seems, from the original site. It's possible now to add all those original archived Newsletters. Can discuss this in the next Skype call. I've also had to remove the Newsletter category from Posts to avoid confusion. 

If you're happy with this approach I'll use it for the Press Releases and the Annual Reports?
